### PR TITLE
adding corporate setup guide for configuring git/ssh form separate accounts

### PR DESCRIPTION
**Pull request by krisnye**
Adding a guide that helps corporate contributors get started.

## ✅ Pull Request Checklist:

- [x] Updated documentation (if it already exists for this component).

## 🧢 Your Team:

Adobe

### CODE REVIEW

File: CORPORATE_SETUP.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+### Step 5: Create a .gitconfig in that directory setting the user to your public email address
+
+`~/Projects/public/.gitconfig`
+
+    [user]
+        email = youremail@gmail.com
+        name = Your Name
+
+### Step 6: Add an [includeIf] section to your ~/.gitconfig telling it to use the new .gitconfig within that directory
```
Comment (snowystinger) [inline CORPORATE_SETUP.md:54]: is this necessary, or does it just make life easier? (asking b/c i've never used that before) in which case maybe we just link to git's documentation on it? or say optional?

File: CORPORATE_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+`~/.gitconfig`
+
+    [user]
+        name = Your Name
+        email = youremail@yourcorporation.com
+    [merge]
+        tool = p4mergetool
+    [mergetool "p4mergetool"]
```
Comment (snowystinger) [inline CORPORATE_SETUP.md:63]: maybe trim this section down, not everyone has access to p4merge outside of adobe

File: CORPORATE_SETUP.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
+
+# Setup Guide for Mac Users with Corporate git credentials.
```
Comment (snowystinger) [inline CORPORATE_SETUP.md:2]: Maybe top heading is Setup Guide for multiple git credentials?
and then second level is Mac
and then continue on
That way we have a space for Windows eventually

File: CORPORATE_SETUP.md (hunk 1 above)
Comment (krisnye) [reply CORPORATE_SETUP.md:54]: I think this is necessary if you're going to use ssh with git: repository urls. If we are just using https: urls to the repo then I can remove quite a few sections related to ssh setup.

File: CORPORATE_SETUP.md (hunk 2 above)
Comment (krisnye) [reply CORPORATE_SETUP.md:63]: can do

File: CORPORATE_SETUP.md (hunk 3 above)
Comment (krisnye) [reply CORPORATE_SETUP.md:2]: sounds good

File: CORPORATE_SETUP.md (hunk 1 above)
Comment (krisnye) [reply CORPORATE_SETUP.md:54]: Step 5 doesn't do anything without Step 6.

**Pull request closed without merging**